### ISSUE TITLE
File name format does not replace wavelength when downloading JSOC images

### ISSUE THREAD

**Issue by wtbarnes** [sunpy/sunpy#4697]
When downloading AIA images from JSOC, the resulting filename format does not include the value of the wavelength, but instead just leaves the  string "WAVELNTH" in the filename. This means that when downloading images at different wavelengths, but the same time interval, the filenames will be the same. This has the effect of tricking `parfive` into not downloading any wavelengths after the first one as the filenames are the same.

I do not know if this is an issue with `drms` or with the way sunpy calls `drms`. Note that the filename format can be specified via the `filenamefmt` kwarg in `drms.Client.export`, but passing this to `Fido.fetch` does not seem to have any effect.

```python
from sunpy.net import Fido,attrs
import astropy.units as u
import astropy.time
t = astropy.time.Time('2012-09-24T14:56:00', scale='utc', format='isot')
q = Fido.search(
    attrs.Time(t, t+13*u.s),
    attrs.jsoc.Wavelength(171*u.angstrom) | attrs.jsoc.Wavelength(171*u.angstrom),
    attrs.Sample(12*u.s),
    attrs.jsoc.Series.aia_lev1_euv_12s,
    attrs.jsoc.Notify('your@email.com'),
    attrs.jsoc.Segment.image,
 )
f = Fido.fetch(q, max_conn=2)
print(f)
```
downloads only one file with the filename `sunpy/data/aia.lev1_euv_12s.2012-09-24T145613Z.WAVELNTH.image.fits` where instead of `WAVELNTH` and the file will be the 94 data while the 171 download is skipped.

sunpy version: master
drms version: 0.6

**Comment by Cadair** [comment]
we don't specify anything to the `filenamefmt` argument to export, should we?

**Comment by wtbarnes** [comment]
If I recall, if I use `drms` directly for the same query and don't specify the `filenamefmt`, this doesn't happen so it seems to be something we are doing...

**Comment by nabobalis** [comment]
This might be fixed upstream by https://github.com/sunpy/drms/pull/52

**Comment by wtbarnes** [comment]
I've just confirmed that this is fixed by sunpy/drms#52

**Comment by dstansby** [comment]
Closing, since the upstream fix is merged
